src/api/createDeviceMessageHandlingSystem/src/v1/steps.py

```diff
@@ -64,32 +139,63 @@ def create_mhs_device(data, cache) -> Device:
     return product.create_device(**device_payload)
 
 
-def create_party_key_tag(data, cache):
+def create_party_key_tag(data, cache) -> DeviceTagAddedEvent:
```

Comment:
party key is supposed to be key rather than a tag, but whilst I'm here... we will need to add a bunch of tags to the device but perhaps best waiting until the bulk etl is done as then the code for doing that will be shareable (I'll write a ticket)

Reply:
Party keys are tags for devices, keys at product level